pkg/controller/v1alpha2/experiment/suggestion/suggestion.go

```diff
@@ -18,6 +17,15 @@ func New() Suggestion {
 	return &General{}
 }
 
-func (g *General) GetSuggestions(instance *experimentsv1alpha2.Experiment, addCount int) ([]*api_pb.Trial, error) {
-	return nil, fmt.Errorf("Not implemented")
+func (g *General) GetSuggestions(instance *experimentsv1alpha2.Experiment, addCount int32) ([]*api_pb.Trial, error) {
+	request := &api_pb.GetSuggestionsRequest{
+		ExperimentName: instance.Name,
+		AlgorithmName:  instance.Spec.Algorithm.AlgorithmName,
+		RequestNumber:  *instance.Spec.ParallelTrialCount,
+	}
+	if reply, err :=common.GetSuggestions(request); err != nil {
```

Comment:
Format here is not gofmted, since the code is merged, we could format it in another PR about suggestion.